<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultimeterExportWizard</class>
 <widget class="QWizard" name="MultimeterExportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <widget class="QWizardPage" name="page_select_datasets">
   <property name="locale">
    <locale language="C" country="AnyCountry"/>
   </property>
   <property name="title">
    <string>Datasets</string>
   </property>
   <property name="subTitle">
    <string>Please select which of the recorded Datasets should be exported.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="list_datasets"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page_select_format">
   <property name="locale">
    <locale language="C" country="AnyCountry"/>
   </property>
   <property name="title">
    <string>Format</string>
   </property>
   <property name="subTitle">
    <string>Please Choose the Fileformat to use</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_formats"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="text_preview"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page_select_file">
   <property name="title">
    <string>Filename</string>
   </property>
   <property name="subTitle">
    <string>Please choose the file to export to.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
